docs/excel/custom-functions-overview.md

```diff
@@ -31,16 +31,16 @@ If you use the [Yo Office generator](https://github.com/OfficeDev/generator-offi
 
 | File | File format | Description |
 |------|-------------|-------------|
-| **./src/functions/functions.js**<br/>or<br/>**./src/functions/functions.ts** | JavaScript<br/>or<br/>TypeScript | Contains the code that defines custom functions. |
-| **./src/functions/functions.json** | JSON | Contains metadata that describes custom functions and enables Excel to register the custom functions and make them available to end users. |
-| **./src/functions/functions.html** | HTML | Provides a &lt;script&gt; reference to the JavaScript file that defines custom functions. |
+| **./src/functions.js**<br/>or<br/>**./src/functions.ts** | JavaScript<br/>or<br/>TypeScript | Contains the code that defines custom functions. |
+| **./config/functions.json** | JSON | Contains metadata that describes custom functions and enables Excel to register the custom functions and make them available to end users. |
+| **./index.html** | HTML | Provides a &lt;script&gt; reference to the JavaScript file that defines custom functions. |
 | **./manifest.xml** | XML | Specifies the namespace for all custom functions within the add-in and the location of the JavaScript, JSON, and HTML files that are listed previously in this table. |
 
 The following sections provide more information about these files.
 
 ### Script file
 
-The script file (**./src/functions/functions.js** or **./src/functions/functions.ts** in the project that the Yo Office generator creates) contains the code that defines custom functions and maps the names of the custom functions to objects in the [JSON metadata file](#json-metadata-file). 
+The script file (**./src/functions.js** or **./src/functions.ts** in the project that the Yo Office generator creates) contains the code that defines custom functions and maps the names of the custom functions to objects in the [JSON metadata file](#json-metadata-file). 
```

Comment:
files *names* are incorrect (should be **customfunctions.js** and **customfunctions.ts**)